cddl/triples-map.cddl

```diff
@@ -15,9 +15,7 @@ triples-map = non-empty<{
     [ + coswid-triple-record ]
   ? &(conditional-endorsement-series-triples: 8) =>
     [ + conditional-endorsement-series-triple-record ]
-  ? &(conditional-endorsement-triples: 9) =>
+  ? &(conditional-endorsement-triples: 10) =>
```

Comment:
it looks like we are doing this half way.

Since we are happy to allocate a new code point, why not picking 11 and deprecating 9 and 10.  While this may not be strictly needed in this case, I think it's a good way to put in place "best practices"

Reply:
@thomas-fossati : it is not like that : code point 10 was already used by MEC, so will continue, using this code point. It just make code point 9 redundant! No need to mint code point 11 here!

Reply:
To summarise:
* On `main`: 9=CET and 10=MEC.
* On `tidyup-triples`: 9=/dev/null and 10=MEC, but it's now called CET 🫨

Reply:
Correct ! Got it 100% right